Makefile

```diff
@@ -16,7 +16,7 @@ PYTHON_EXE ?= python
 # (introduced in python 2.6)
 DISTUTILS_PLAT_NAME = $(shell $(PYTHON_EXE) setup.py --help build_ext \
 		      | grep -q -- "--plat-name" && echo "--plat-name win32")
-PYTHON_CHECK_STYLE_TARGETS = pycam Tests pyinstaller/hooks/hook-pycam.py scripts/pycam setup.py
+PYTHON_CHECK_STYLE_TARGETS = pycam pyinstaller/hooks/hook-pycam.py scripts/pycam setup.py
```

Comment:
OK, I see what we're doing here, and maybe this isn't the right venue, BUT...

A few thoughts:

   *  do we still need to support svn?
   *  can we start migrating test/build/deployment infrastructure from make, into the `.travis.yml`?
   *  how are we populating this list of targets?  Is it growing as we start managing more elements of the codebase with style checks?